Simplenote/Classes/SPSettingsViewController+Extensions.swift

```diff
@@ -1,16 +1,36 @@
 import UIKit
 
 
-// MARK: - Interface Helpers
+// MARK: - Subscriber UI
 //
 extension SPSettingsViewController {
 
+    private var isActiveSustainer: Bool {
+        SPAppDelegate.shared().simperium.preferencesObject().isActiveSubscriber
```

Comment:
The `preferencesObject` getter ensures that if such object wasn't available, it'll be inserted. This is expected to default to `false`